packages/resolve-bus-memory/src/index.js

```diff
@@ -0,0 +1,34 @@
+const eventHandlersMap = new Map();
```

Comment:
Because of existence memory bus in only one process, it's same - all events are routed by callback reference. So in this approach message bus is singleton by design,